content/en/docs/concepts/scheduling-eviction/pod-overhead.md

```diff
@@ -105,7 +105,7 @@ When the kube-scheduler is deciding which node should run a new Pod, the schedul
 requests and the overhead, then looks for a node that has 2.25 CPU and 320 MiB of memory available.
 
 Once a Pod is scheduled to a node, the kubelet on that node creates a new {{< glossary_tooltip
-text="cgroup" term_id="cgroup" >}} for the Pod. It is within this pod that the underlying
+text="cgroup" term_id="cgroup" >}} for the Pod. It is within this cgroup that the underlying
```

Comment:
```suggestion
text="cgroup" term_id="cgroup" >}} for the Pod. It is within this cgroup that the underlying
```
The proposed change of this word from pod to cgroup is tricky since a container isn't necessarily created within a cgroup but with the use of cgroups and namespaces (uts, network, pid, user, ipc)

My previous lgtm is from a SIG Docs view
It would be great if have a technical review from @kubernetes/sig-node-pr-reviews or a sme like @mikebrow

Reply:
I'm not sure the rewording here is super helpful. For most people's mental model, I think keeping this as `pod` probably makes more sense.

Reply:
quick skim to this point of the Pod Overhead documentation...

The ```{{< glossary_tooltip
 text="cgroup" term_id="cgroup" >}}``` should also be included above on line 28 where it is initially referenced 

```suggestion
text="cgroup" term_id="cgroup" >}} with the resource limits calculated for the Pod. It is within the pod's cgroup limits that the underlying container runtime creates the pod and it's containers.
```

Reply:
I agree with @endocrimes 's [comment ](https://github.com/kubernetes/website/pull/34337#discussion_r930902759) to keep the word `pod` instead in line 108